x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java

```diff
@@ -1230,8 +1229,8 @@ private void assertHistoryIsPresent(
             fail("failed to perform search:" + e.getMessage());
         }
 
-        // Finally, check that the history index is in a good state
-        String historyIndexName = DataStream.getDefaultBackingIndexName("ilm-history-7", 1);
+        String historyIndexName = getDataStreamBackingIndexNames("ilm-history-7").getFirst();
+        assertThat(historyIndexName, notNullValue());
```

Comment:
Nit: I don't think this assertion is super valuable. The only way it could be null is if the GET data streams API returns the data stream, with indices, but the index doesn't have it's `index_name` set -- in all other cases, `getDataStreamBackingIndex` will throw an exception already.

Reply:
Good point I will remove it.